src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
It should be required, it doesn't make sense to render user info without a user